constants/config.ts

```diff
@@ -22,3 +22,24 @@ export const HUBSPOT_ACCOUNT_TYPE_STRINGS = {
   APP_DEVELOPER: i18n('lib.accountTypes.appDeveloper'),
   STANDARD: i18n('lib.accountTypes.standard'),
 } as const;
+
+export const CONFIG_FLAGS = {
+  USE_CUSTOM_OBJECT_HUBFILE: 'useCustomObjectHubfile',
+} as const;
+
+export const ENVIRONMENT_VARIABLES = {
+  HUBSPOT_API_KEY: 'HUBSPOT_API_KEY',
+  HUBSPOT_CLIENT_ID: 'HUBSPOT_CLIENT_ID',
+  HUBSPOT_CLIENT_SECRET: 'HUBSPOT_CLIENT_SECRET',
+  HUBSPOT_PERSONAL_ACCESS_KEY: 'HUBSPOT_PERSONAL_ACCESS_KEY',
+  HUBSPOT_ACCOUNT_ID: 'HUBSPOT_ACCOUNT_ID',
+  HUBSPOT_PORTAL_ID: 'HUBSPOT_PORTAL_ID',
+  HUBSPOT_REFRESH_TOKEN: 'HUBSPOT_REFRESH_TOKEN',
+  HUBSPOT_ENVIRONMENT: 'HUBSPOT_ENVIRONMENT',
+  HTTP_TIMEOUT: 'HTTP_TIMEOUT',
+  HTTP_USE_LOCALHOST: 'HTTP_USE_LOCALHOST',
+  ALLOW_USAGE_TRACKING: 'ALLOW_USAGE_TRACKING',
+  DEFAULT_CMS_PUBLISH_MODE: 'DEFUALT_CMS_PUBLISH_MODE',
+  USE_ENVIRONMENT_CONFIG: 'USE_ENVIRONMENT_CONFIG',
```

Comment:
Slight nit, but does it make sense for `USE_ENVIRONMENT_CONFIG` to include "hubspot" in the name somewhere? It doesn't matter a ton if the CLI is setting it, but I imagine people can also manually set this if they want to.